src/main/java/net/openhft/chronicle/bytes/Bytes.java

```diff
@@ -635,19 +635,22 @@ static String toString(@NotNull final Bytes<?> buffer,
     }
 
     /**
-     * Extracts a string from the provided {@code buffer} starting at the specified {@code position},
+     * <p>Extracts a string from the provided {@code buffer} starting at the specified {@code position},
      * and spanning for the specified {@code length} number of characters. The buffer's state
-     * remains unchanged by this method.
-     * <p>
-     * The method reads {@code length} bytes from the {@code buffer}, starting at {@code position},
-     * and constructs a string from these bytes.
+     * remains unchanged by this method.</p>
+     *
+     * <p>The method reads {@code length} bytes from the {@code buffer}, starting at {@code position},
+     * and constructs a string from these bytes.</p>
+     *
+     * <p>This method supports all characters in the Basic Latin Unicode block.</p>
      *
      * @param buffer   The buffer to extract the string from. Must not be {@code null}.
      * @param position The position in the buffer to start extracting the string from.
      * @param length   The number of characters to include in the extracted string.
      * @return A string extracted from the buffer.
      * @throws IllegalArgumentException If the provided {@code position} or {@code length} is negative.
      * @throws NullPointerException     If the provided {@code buffer} is {@code null}.
+     * @see <a href="https://en.wikipedia.org/wiki/Basic_Latin_(Unicode_block)" target="_blank">Basic Latin Unicode block</a>
```

Comment:
love it

Reply:
please mention that it won't handle the upper block and refer to Bytes.to8BitString

Reply:
Done!